### ISSUE TITLE
Dataframe.sum() not recognising `min_count` parameter

### ISSUE THREAD

**Issue by Rendiere** [dask/dask#3981]
According to the [documentation](http://dask.pydata.org/en/latest/dataframe-api.html#dask.dataframe.DataFrame.sum), the `Dataframe.sum()` method accepts a `min_count` parameter which controls the threshold for minimum acceptable number of NaNs in the sum.

However, this parameter does not seem to be currently recognised (although present even in docstrings). 

Below is a simple example to reproduce this error:

```python
df = pd.DataFrame({'x': [1, 2, 3, np.nan, 5, np.nan, np.nan, 8, 9, 10],
                 'y': [1, 2, 3, np.nan, 5, np.nan, np.nan, 8, 9, 10],
                 'z':[1, 2, 3, np.nan, 5, np.nan, 7, 8, 9, 10]})

ddf = dd.from_pandas(df, npartitions=2)

nan_sum = ddf.sum(axis=1, min_count=1).compute()

```

Which throws the error:

```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-15-9bb76e49f32d> in <module>()
----> 1 ddf.sum(axis=1, min_count=1)

TypeError: sum() got an unexpected keyword argument 'min_count'
```

**Comment by TomAugspurger** [comment]
The docstrings are auto-generated form pandas, and the `Notes` field notes that it's not supported.

Supporting it should be possible, if you're interested in working on it. Let us know if you need help getting started.

**Comment by TomAugspurger** [comment]
We’ll leave this open to properly support it.

**Comment by jrbourbeau** [comment]
Closed by #4090
